src/components/News/News.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Article } from "../../types/article";
+import { NewsArticle } from "../NewsArticle";
+import styles from "./News.module.css";
+
+interface NewsProps {
+  articles: Array<Article>;
+}
+
+export const News = ({ articles }: NewsProps) => {
+  return (
```

Comment:
Need to add a button for `Add New Article` to the header (don't need to wire it up yet).